static/index.js

```diff
@@ -14,10 +14,10 @@ let isShowingInfo = false;
 const navConfig = {
     default: [
         { src: "/static/icons/ai.svg", alt: "AI Tags", class: "toggle", onclick: "toggleGroup(this)" },
```

Comment:
@dexterousdhruv

Reply:
Okay! I'll do that.

Reply:
@dexterousdhruv 
This one is yet to be resolved.

Except for elements sourced from default, the IDs of other elements are getting set to undefined. 

Either assign unique IDs to all elements 
**or** 
Generate IDs from the alt attribute (ensuring to handle spaces appropriately), rather than specifying them manually